src/controllers/TaskController.ts

```diff
@@ -50,4 +50,17 @@ export class TaskController {
       return res.status(500).send({ message: 'Internal Server Error' });
     }
   }
+
+  async getTasks(req: Request, res:Response) {
+    try {
+      const tasks = await this.taskCollection.find({}).toArray();
+      if (!tasks) {
+        return res.status(404).send({ message: 'Tasks not found' });
```

Comment:
```suggestion
        return res.status(204).send({ message: 'Task list is empty' });
```
  404 oznacza blad, ze nie udalo nam sie czegos znalezc, np szukamy usera, a baza mowi, ze nie ma takiego. U ciebie to, ze lista jest pusta nie oznacza bledu:[ http-codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status#successful_responses:~:text=to%20this%20status.-,204%20No%20Content,-There%20is%20no)

Reply:
ewentualnie mozesz tego if'a w ogole usunac i jak bedzie pusto to w returnie w 60 linii zwrocisz po prostu pusta tablice co tez jest poprawne (w zasadzie bardziej niz rozwiazanie wyzej)